### PR TITLE
Python upgrade

### PR DESCRIPTION
**Pull request by eugenevinitsky**
Bump python version in environment file.

### CODE REVIEW

File: examples/exp_configs/rl/multiagent/multiagent_i210.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "human",
     num_vehicles=0,
     lane_change_params=SumoLaneChangeParams(lane_change_mode="strategic"),
-    acceleration_controller=(IDMController, {"a": .3, "b": 2.0, "noise": 0.6}),
+    acceleration_controller=(IDMController, {"a": .3, "b": 2.0, "noise": 0.5}),
```
Comment (kanaadp) [inline examples/exp_configs/rl/multiagent/multiagent_i210.py]: different PR pls

File: flow/controllers/base_controller.py [hunk 2]
```diff
@@ -107,7 +107,7 @@ def get_action(self, env):
 
         # add noise to the accelerations, if requested
         if self.accel_noise > 0:
-            accel += np.random.normal(0, self.accel_noise)
+            accel += np.sqrt(env.sim_step) * np.random.normal(0, self.accel_noise)
```
Comment (kanaadp) [inline flow/controllers/base_controller.py]: same

File: flow/envs/multiagent/i210.py [hunk 3]
```diff
@@ -144,7 +144,8 @@ def compute_reward(self, rl_actions, **kwargs):
                 rewards[rl_id] = 0
                 speeds = []
                 follow_speed = self.k.vehicle.get_speed(self.k.vehicle.get_follower(rl_id))
-                speeds.extend([speed for speed in follow_speed if speed >= 0])
+                if follow_speed >= 0:
```
Comment (kanaadp) [inline flow/envs/multiagent/i210.py]: same as above

**Comment by eugenevinitsky** [comment]
@kanaadp this should all be in a different PR

File: examples/exp_configs/rl/multiagent/multiagent_i210.py (hunk 1 above)
Comment (eugenevinitsky) [reply examples/exp_configs/rl/multiagent/multiagent_i210.py]: I don't understand, these changes are already merged into the other branch

**Pull request merged**